deploy/template.yaml

```diff
@@ -128,6 +128,51 @@ Mappings:
 
 Resources:
 
+  # ssl cert
+  DevSSLCert:
+    Type: AWS::CertificateManager::Certificate
+    Condition: IsNewDevelopment
+    Properties:
+      DomainName: !Sub "${Environment}.dev.identity.account.gov.uk"
+      DomainValidationOptions:
+        - DomainName: !Sub "${Environment}.dev.identity.account.gov.uk"
+          HostedZoneId: !ImportValue DevIdentityHostedZoneId
+      ValidationMethod: DNS
+
+  # api domain entries / mapping
+  DevApiDomain:
+    Type: AWS::ApiGatewayV2::DomainName
+    Condition: IsNewDevelopment
+    Properties:
+      DomainName: !Sub "${Environment}.dev.identity.account.gov.uk"
```

Comment:
Wondering if this should be "$core-front-{Environment}.dev.identity.account.gov.uk" or "$core-{Environment}.dev.identity.account.gov.uk" just to be super obvious when running journeys what you're looking at.

Reply:
Discussed offline that it's fine as is. Should be clear from context what you're looking at.